### PR TITLE
Add sessionObject model  

### PR DESCRIPTION
**Pull request by glushko311**
Add sessionObject model  and request to create table in migration.py

### CODE REVIEW

File: src/app/migrate.py [hunk 1]
```diff
@@ -32,8 +32,18 @@
                               "                    comment TEXT, \n"
                               "                    CONSTRAINT passwords_pk PRIMARY KEY (pass_id) \n"
                               "                    ) WITH OIDS;")
+SQL_create_table_session_objects = ("CREATE TABLE session_objects ( \n"
```
Comment (VanyaDNDZ) [inline src/app/migrate.py]: Why are you using raw SQL?

File: src/app/models.py [hunk 2]
```diff
@@ -121,3 +121,42 @@ def __init__(self, login, password, user_id, url, title, comment):
...
+class SessionObject(Base):
+    __tablename__ = 'session_objects'
+
+    id = Column('id', Integer, primary_key=True)
+    token = Column('token', String(100), unique=True, nullable=False)
+
+    user_id = Column('user_id', Integer, ForeignKey('users.id'))
+    user = relationship("User", backref="session_objects", cascade='all,delete')
+
+    login_time = Column('login_time', Date, nullable=False)
```
Comment (VanyaDNDZ) [inline src/app/models.py]: Date type not valid for a timeout check. What if your session is valid for 30 minutes?

File: src/app/migrate.py (hunk 1 above)
Comment (glushko311) [reply src/app/migrate.py]: This part(migrate.py) will be refactored by Vitaly. I use raw SQL only as temporary solution.

**Comment by glushko311** [comment]
Fix type Date on type DateTime in the login_time field in the objectSession model

File: src/app/migrate.py (hunk 1 above)
Comment (VanyaDNDZ) [reply src/app/migrate.py]: Ok

**Pull request merged**